packages/components/src/RecurringSelect/components/MonthlyDayOfWeekSelect/MonthlyDayOfWeekSelect.tsx

```diff
@@ -2,7 +2,8 @@ import React from "react";
 import styles from "./MonthlyDayOfWeekSelect.css";
 import { Text } from "../../../Text";
 import { WeekDay } from "../../types";
-import { WeeklySelect } from "../index";
+// eslint-disable-next-line
+import { WeeklySelect } from "../WeeklySelect/WeeklySelect";
```

Comment:
tried to change this to `from "../index";` where we are exporting the components from `index.ts`

this allowed me to remove the eslint disable rule, which seems preferable

the issue there is that we then get a https://github.com/import-js/eslint-plugin-import/blob/v2.18.2/docs/rules/no-cycle.md

on the `export { MonthlyDayOfWeekSelect } from "./MonthlyDayOfWeekSelect/MonthlyDayOfWeekSelect";
` line in `components/index.tsx`

not exactly sure what the best course of action is here. we can either disable this and live with it, or we can stop exporting from the `index.ts` perhaps, but that means disabling eslint on a few other imports at which point if we're comparing 1 eslint disable vs plural, that seems like a clear cut winner to me

Reply:
Aside from the real issue of cycle dependency, this is basically a sign that the file structure combined with imports needs a bit of work